models/orb/orb-v3.yml

```diff
@@ -0,0 +1,170 @@
+model_name: ORB v3
+model_key: orb-v3
+model_version: v3
+matbench_discovery_version: 1.3.1
+date_added: "2025-04-05"
+date_published: "2025-04-05"
+authors:
+  - name: Ben Rhodes
+    affiliation: Orbital Materials
+    corresponding: true
+  - name: Sander Vandenhaute
+    affiliation: Orbital Materials
+  - name: Vaidotas Šimkus
+    affiliation: Orbital Materials
+  - name: James Gin-Pollock
+    affiliation: Orbital Materials
+  - name: Tim Duignan
+    affiliation: Orbital Materials
+  - name: Jonathan Godwin
+    affiliation: Orbital Materials
+    email: jonathan@orbitalmaterials.com
+  - name: Mark Neumann
+    affiliation: Orbital Materials
+    email: mark@orbitalmaterials.com
+    corresponding: true
+
+repo: https://github.com/orbital-materials/orb-models
+url: https://github.com/orbital-materials/orb-models
+doi: https://github.com/orbital-materials/orb-models
+paper: https://github.com/orbital-materials/orb-models
+pr_url: https://github.com/janosh/matbench-discovery/pull/240
+checkpoint_url: https://orbitalmaterials-public-models.s3.us-west-1.amazonaws.com/forcefields/orb-v3/orb-v3-conservative-inf-mpa-20250404.ckpt
+
+license:
+  code: Apache-2.0
+  code_url: https://github.com/orbital-materials/orb-models/blob/3e9580809/LICENSE
+  checkpoint: Apache-2.0
+  checkpoint_url: https://github.com/orbital-materials/orb-models/blob/3e9580809/LICENSE
+
+openness: OSCD
+trained_for_benchmark: true
+train_task: S2EFS
+test_task: IS2RE-SR
+targets: EFS_D
```

Comment:
Is this model conservative? EFS_D is EFS direct but the URL suggests it's conservative therefore EFS_G would be the right thing here

Reply:
Ah, yep! will fix